tcms/locale/de_DE/LC_MESSAGES/django.po

```diff
@@ -124,6 +139,10 @@ msgstr "Diese Aktivierungs-Code ist abgelaufen"
 msgid "Your account has been activated successfully"
 msgstr "Dein Konto wurde erfolgreich aktiviert"
 
+#: tcms/management/tests/tests.py:76
+msgid "Yes, I'm sure"
```

Comment:
These are from recent changes I did and there are more coming after merging #802. I am regenerating the source strings right now and you will probably get a conflict.

While this PR is still in progress you can drop this commit and return to this at the end before we merge.